### DISCUSSION TITLE
Released v3.0.0-beta

### DISCUSSION THREAD

**Discussion by imzbf** [imzbf/md-editor-v3#240] [Announcements]
`beta`版本已发布，欢迎试用~

```bash
npm i md-editor-v3@beta

# or 

yarn add md-editor-v3@beta
```

在`3.0.0`中

- 使用[codemirror](https://codemirror.net/)替换了默认的`textarea`编辑区
- 使用[markdown-it](https://github.com/markdown-it/markdown-it)替换了核心编译库`marked`

因为核心替换，`config`方法现在支持`markdown-it`和`codemirror`的扩展，移除了原`marked`的相关扩展项。

正式版及文档未开发完成，使用[基础参考](https://github.com/imzbf/md-editor-v3/tree/v3)

<img width="1185" alt="image" src="https://user-images.githubusercontent.com/37092006/231402390-50fe3609-e571-4dec-bbf5-c7e4a7db5629.png">

**Comment by daidaibg** [comment]
感觉这次更新感觉很棒

**Reply by imzbf** [reply]
希望会很棒，带来的问题也不少😄
